docusaurus/docs/tutorials/build-a-streaming-data-source-plugin.md

```diff
@@ -79,7 +79,7 @@ export function QueryEditor({ query, onChange, onRunQuery }: Props) {
   const { upperLimit, lowerLimit, tickInterval } = query;
 
   return (
-    <div className="gf-form">
+    <>
```

Comment:
Could potentially use the new `Box` component here, but it does not change appearance so don't think we need it.